rkt/install.go

```diff
@@ -133,8 +134,8 @@ func lookupGid(groupName string) (gid int, err error) {
 	return group.Gid, nil
 }
 
-func setDirPermissions(path string, uid int, gid int, perm os.FileMode) error {
-	if err := os.Chown(path, 0, gid); err != nil {
+func setPermissions(path string, uid int, gid int, perm os.FileMode) error {
```

Comment:
Just thought the function that way (and maybe reusage) but yeah, it could be removed